src/lazyfree.c

```diff
@@ -207,18 +207,6 @@ void emptyDbAsync(redisDb *db) {
     bioCreateLazyFreeJob(lazyfreeFreeDatabase,2,oldht1,oldht2);
 }
 
-/* Release the radix tree mapping Redis Cluster keys to slots.
- * If the rax is huge enough, free it in async way. */
-void freeSlotsToKeysMapAsync(rax *rt) {
-    /* Because this rax has only keys and no values so we use numnodes. */
-    if (rt->numnodes > LAZYFREE_THRESHOLD) {
-        atomicIncr(lazyfree_objects,rt->numele);
-        bioCreateLazyFreeJob(lazyfreeFreeSlotsMap,1,rt);
```

Comment:
lazyfreeFreeSlotsMap(It's up at the top of this file) is dead code now, so please clean it up